libunwind/src/UnwindCursor.hpp

```diff
@@ -2974,6 +2966,37 @@ bool UnwindCursor<A, R>::getFunctionName(char *buf, size_t bufLen,
                                          buf, bufLen, offset);
 }
 
+#if defined(_LIBUNWIND_CHECK_LINUX_SIGRETURN)
+template <typename A, typename R>
+bool UnwindCursor<A, R>::isReadableAddr(const pint_t addr) const {
+  // This code is heavily based on Abseil's 'address_is_readable.cc',
+  // which is Copyright Abseil Authors (2017).
```

Comment:
I think it is fine to just say without more quote about the license.

We use SYS_rt_sigprocmask, inspired by Abseil's AddressIsReadable.

Reply:
Done